### PR TITLE
hotel rendering bug

### PR DESCRIPTION
**Pull request by Kumjami**
fix bug that cause not to render packages while some amenities are boolean instead of strings

### CODE REVIEW

File: src/components/hotel/index.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ class HotelPage extends Component {
     for (var key in amenitiesObject) {
       if (!amenitiesObject.hasOwnProperty(key)) continue;
       if (key === fact) {
-        if (amenitiesObject[key] === typeof boolean || amenitiesObject[key] === 'true' || amenitiesObject[key] === 'false') {
+        if (typeof amenitiesObject[key] === 'boolean' || amenitiesObject[key] === 'true' || amenitiesObject[key] === 'false') {
```
Comment (lennym) [inline src/components/hotel/index.js]: Given that all the other conditions assume the value type to be `string`, would it be safer to check `typeof amenitiesObject[key] !== 'string'`?

Comment (Kumjami) [inline src/components/hotel/index.js]: Then 'true' or 'false' would be evaluated as a string, not as boolean.

Comment (lennym) [inline src/components/hotel/index.js]: I meant in addition to the `=== 'true'` and `=== 'false'` clauses, not in place of the entire condition. So the specific `'true'` and `'false'` values would still be matched.

i.e. 

```javascript
if (typeof amenitiesObject[key] !== 'string' || amenitiesObject[key] === 'true' || amenitiesObject[key] === 'false') {
...
}
```

**Comment by nelsonic** [comment]
@Kumjami please assign this PR for review when ready. 👍

**Comment by nelsonic** [comment]
merging. 👍

**Pull request merged**